glib/src/types.rs

```diff
@@ -246,6 +246,19 @@ impl StaticType for Type {
     }
 }
 
+pub trait StaticTypeExt {
+    // rustdoc-stripper-ignore-next
+    /// Ensures that the type has been registered with the type system.
+    #[doc(alias = "g_type_ensure")]
+    fn type_ensure();
```

Comment:
`gtk::Button::type_ensure()` doesn't read very nice. Maybe this should be `gtk::Button::ensure_type()`?